module/icp/include/sha2/sha2_impl.h

```diff
@@ -57,6 +57,11 @@ typedef struct sha2_hmac_ctx {
 	SHA2_CTX		hc_ocontext;	/* outer SHA2 context */
 } sha2_hmac_ctx_t;
 
+/*
+ * Initialize sha2 implementations.
+ */
+void sha2_impl_init(void);
```

Comment:
Is this considered an ABI change?

Reply:
It looks like it, but for `libzpool` and the kmods we don't enforce a stable ABI so this is OK.